src/cmd/shared.go

```diff
@@ -16,6 +16,7 @@ var (
 	debugFlag,
 	dryRunFlag,
 	globalFlag bool
+	repoCache *git.ProdRepo
```

Comment:
Thoughts on making this a const called `repo` that is initialized? Not sure if the constructor has side effects we don't want to run at other times

Reply:
Though I guess that may introduce confusion as repo is a local variable in many functions in this package. Maybe call it `prodRepo` then?

Reply:
Great idea! All commands need this variable anyways. I went with `prodRepo`.